### PR TITLE
webview: attribute to disable web notifications

### LINKED ISSUE #4174: webview permission-request event
Would it be possible to disable notifications for a webview? I know you can add a preload script to delete window.Notification, but if an iframe tries to send a notification this still succeeds.

### PR DESCRIPTION
**Pull request by deepak1556**
Fixes #4174 

Depends on https://github.com/atom/brightray/pull/193

### CODE REVIEW

File: atom/browser/atom_browser_client.h [hunk 1]
```diff
@@ -81,6 +81,7 @@ class AtomBrowserClient : public brightray::BrowserClient,
   // brightray::BrowserClient:
   brightray::BrowserMainParts* OverrideCreateBrowserMainParts(
       const content::MainFunctionParams&) override;
+  bool WebNotificationAllowed(int render_process_id) override;
```
Comment (zcbenz) [inline atom/browser/atom_browser_client.h]: I didn't find this method in `content_browser_client.h`?

**Comment by zcbenz** [comment]
I'm strongly against adding flag for every web feature, otherwise we are going to end up doubling the number of options in `webContents` and `<webview>`. Something like webview's permissionrequest event is more suitable for this.

**Comment by deepak1556** [comment]
yeah you are right, will reopen once implemented.

**Comment by saenzramiro** [comment]
Any news on this? :+1:

**Comment by deepak1556** [comment]
@saenzramiro this was handled in https://github.com/electron/electron/pull/4223

**Pull request closed without merging**